_layouts/candidate.html

```diff
@@ -89,13 +92,48 @@ <h2 class="candidate__money-heading">Other committees controlled by this candida
       </div>
     </section>
     {% endif %}
-    {% if opposing_committees and opposing_committees != empty %}
+    {% if expenditures_opposing > 0 or expenditures_supporting > 0 %}
     <section class="l-section">
       <div class="l-section__content">
-        <h2 class="candidate_money-heading">Committees opposing candidate</h2>
-        {% for committee in opposing_committees %}
-        <p><a href="{{ site.baseurl }}/committee/{{ committee.filer_id }}/">{{ committee.name | smartify | escape }}</a></p>
-        {% endfor %}
+        {% if expenditures_opposing > 0 %}
+        <div class="candidate__expenditures">
+          Independent expenditures opposing candidate <span class="money">{{ expenditures_opposing | dollars }}</span>
+          <span class="hover-info-container"><img src="{{ '/assets/images/icon_more_info.png' | relative_url }}"
+            alt="Independent expenditures - more information" />
+            <span class="hover-info">Spending by third parties that advocates the election or defeat of a candidate and is not made in coordination with a candidate or campaign committee is termed an independent expenditure. To learn more,
+              <a href="{{ site.baseurl }}/faq/">see the FAQ </a>.</span>
+          </span>
+        </div>
+        {% endif %}
+        {% if expenditures_supporting > 0 %}
+        <div class="candidate__expenditures">
+          Independent expenditures supporting candidate <span class="money">{{ expenditures_supporting | dollars }}</span>
+          {% if expenditures_opposing <= 0 %}
+          <span class="hover-info-container"><img src="{{ '/assets/images/icon_more_info.png' | relative_url }}"
+            alt="Independent expenditures - more information" />
+            <span class="hover-info">Spending by third parties that advocates the election or defeat of a candidate and is not made in coordination with a candidate or campaign committee is termed an independent expenditure. To learn more,
+              <a href="{{ site.baseurl }}/faq/">see the FAQ </a>.</span>
+          </span>
```

Comment:
Same, this could also be captured by the new include.

Reply:
ca156d7